packages/shared/routes/dashboard/staking/views/StakingInfo.svelte

```diff
@@ -27,7 +27,7 @@
             animation = null
         } else if ($stakingEventState === ParticipationEventState.Holding) {
             const stakingParticipationIds: string[] = []
-            $selectedAccountParticipationOverview.participations.forEach((p) => {
+            $selectedAccountParticipationOverview?.participations.forEach((p) => {
```

Comment:
to be super safe, perhaps we could also add another one? 
```suggestion
            $selectedAccountParticipationOverview?.participations?.forEach((p) => {
```